mmv1/third_party/tgc/tests/data/example_google_dataflow_job.json

```diff
@@ -0,0 +1,19 @@
+[
+    {
+        "name": "//dataflow.googleapis.com/v1b3/projects/{{.Provider.project}}/locations/placeholder-2EsS2bm9/jobs",
```

Comment:
Can you please verify the name format from the CAI representation returned from ListAssets?

I think "v1b3" is not required in the name

Reply:
done, thanks